src/icons/index.ts

```diff
@@ -69,6 +69,7 @@ export { default as TriangleIcon } from './triangle.svg';
 export { default as TryAgainIcon } from './try-again.svg';
 export { default as WarningIcon } from './warning.svg';
 export { default as WithdrawIcon } from './withdraw.svg';
+export { default as DownloadIcon } from './download.svg';
```

Comment:
nit: alphabetize

Reply:
hmm interesting, we have organize imports on save enabled in our .vscode settings. @DavideSegullo what IDE are y'all using to make updates?

Reply:
I use vscode

Reply:
weird... but not a big deal

Reply:
Done